internal/dag/dag.go

```diff
@@ -365,11 +384,14 @@ type Route struct {
 	// If false, traffic is allowed only if it doesn't match any rule.
 	IPFilterAllow bool
 
-	// IPFilterRules is a list of ipv4/6 filter rules for which matching
+	// IPFilterRules i /6 filter rules for which matching
```

Comment:
```suggestion
	// IPFilterRules is a list of ipv4/6 filter rules for which matching
```
There seems to be a mistake; I’m not sure why you changed it...